CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.149.0] - 2021-07-20
```

Comment:
You don't need to bump the version manually, just remove this line and it will be ok. 
```suggestion
```